### DISCUSSION TITLE
Consistency with the “weak vowel merger”

### DISCUSSION THREAD

**Discussion by trosel** [Shavian-info/readlex#39] [General]
I noticed that “secret” is spelled like 𐑕𐑰𐑒𐑮𐑩𐑑 which exhibits a “weak vowel merger”.

https://en.m.wikipedia.org/wiki/Phonological_history_of_English_close_front_vowels#Weak_vowel_merger

 However “rabbit” is spelled like 𐑮𐑨𐑚𐑦𐑑 which does not exhibit this same weak vowel merger. 

Wiktionary here: https://en.m.wiktionary.org/wiki/secret
Cambridge here: https://dictionary.cambridge.org/dictionary/english/secret?q=Secret

notice how the Cambridge IPA data shows the weak vowel merger, but the *audio* for both UK and US shows a clear 𐑦𐑑 more like the word “rabbit” and not the schwa Weak Vowel. 

I believe that the readlex standard should seek to be consistent in how it lands on either side of this weak vowel merger. 

Thoughts?

**Comment by Shavian-info** [comment]
You've hit on an area of some uncertainty to me. You'll see in the ReadLex raw data that I mark some of these cases with a capital Ə in cases where Shavian uses schwa by convention (distinct from RP) and capital I in other more edge cases. The rules of thumb I apply are 1) follow Androcles - this means for example that the superlative ending '-est' becomes 𐑩𐑕𐑑, while final 'est' in other words becomes 𐑦𐑕𐑑 (so, 𐑞 𐑛𐑸𐑒𐑩𐑕𐑑 𐑓𐑪𐑮𐑦𐑕𐑑). 2) With -et words I tend to spell the diminutive meaning 𐑩𐑑 and otherwise follow the dictionaries which most often use 𐑦𐑑.  So 𐑑𐑦𐑒𐑩𐑑 but also 𐑮𐑨𐑚𐑦𐑑. Again, this is inspired by Androcles. It's not perfect but it gives a sort of consistency.

**Reply by trosel** [reply]
I notice that the Readlex standard tends to err on the side of the schwa in pretty much all cases. 

For me, I would prefer to do the opposite and err on the side of clarity. Only do a schwa where it is obviously the vowel. Eg. “Water” the second vowel is a schwa. 

But piglet, rabbit, secret, I think it would be more clear to not reduce these to a schwa. Erring on the side of not reducing the vowel would also potentially increase diversity of sound and the recognition (uniqueness) of words. 

I would definitely say -Et and -est with the “in” letter. But I appreciate that the readlex is at least trying to be predictable 

By the way, what you are describing is what I was hitting on here as well https://github.com/Shavian-info/readlex/discussions/32
The Readlex already seems to favor consistency over accurate pronunciation

**Reply by trosel** [reply]
I believe that the CUBE dictionary follows your current system in Readlex. http://cube.elte.hu/

**Reply by trosel** [reply]
Update, I noticed that you use “ago” for bracket, but CUBE, Cambridge, and Wiktionary use “in”. 

Bracket is technically the diminutive according to etymology but everything seems to be clear that it is not “ago” for the final syllable. 

I’m not sure that it is possible to apply “ago” unilaterally for all diminutives unless when we teach the letter “ago”, we explain that sometimes this letter takes on an “in” sound in the final syllable. But given how different “ago” and “in” are, that can’t really be justified as being simply a flexible phoneme. We would simply be losing the accurate spelling that shavian provides.

**Reply by Shavian-info** [reply]
I'm sticking with the Androcles approach for now (see 'basket' on pp. 122-123 of Androcles). Pronunciations are too variable and indistinct across accents simply to adopt a case-by-case 'follow the pronunciation' approach. And the Androcles orthography is (in)famous for its 'Rosa's roses' merger, which shows that for weak vowels it does tend to err on the side of 𐑩 more than RP and General American, but far less than southern hemisphere Englishes.

**Reply by trosel** [reply]
Fair enough, I just noticed that if CUBE is showing “in”, it could be a good indication that even the typical British accent has changed since Androcles. Would be interested in other peoples’ feedback on this. 

There has to be some halfway solution between consistency of writing/grammar and accuracy of sound. Maybe it lies in breaking it apart further than just diminutive suffix and not diminutive suffix
